Frizz/OfficialMenu.swift

```diff
@@ -0,0 +1,200 @@
+//
+//  OfficialMenu.swift
+//  Frizz
+//
+//  Created by Reem Amer on 8/5/18.
+//  Copyright © 2018 Building Something Cool. All rights reserved.
+//
+
+import UIKit
+
+enum Colors {
+    
+    static let red = UIColor(red: 1.0, green: 0.0, blue: 77.0/255.0, alpha: 1.0)
+    static let blue = UIColor.blue
+    static let green = UIColor(red: 35.0/255.0 , green: 233/255, blue: 173/255.0, alpha: 1.0)
+    static let yellow = UIColor(red: 1, green: 209/255, blue: 77.0/255.0, alpha: 1.0)
+}
+
+enum Images {
+    
+    static let box = UIImage(named: "Box")!
+    static let triangle = UIImage(named: "Triangle")!
+    static let circle = UIImage(named: "Circle")!
+    static let swirl = UIImage(named: "Spiral")!
+}
+
+class OfficialMenu: UIViewController, PaginingDelegate {
+    
+    var name: String?
+    var phone: String?
+    var pages = UIPageControl()
+    var emitter = CAEmitterLayer()
+    var colors:[UIColor] = [
+        Colors.red,
+        Colors.blue,
+        Colors.green,
+        Colors.yellow
+    ]
+    var images:[UIImage] = [
+        Images.box,
+        Images.triangle,
+        Images.circle,
+        Images.swirl
+    ]
+    var velocities:[Int] = [
+        100,
+        90,
+        150,
+        200
+    ]
+    
+    let nameLabelView: UILabel = {
+        let nameView = UILabel()
+        nameView.font = UIFont.boldSystemFont(ofSize: 20)
+        nameView.font = UIFont(name: "Avenir-Roman", size: 30)
+        nameView.textAlignment = .left
+        nameView.textColor = .black
+        nameView.translatesAutoresizingMaskIntoConstraints = false
+        return nameView
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        view.addSubview(nameLabelView)
+        
+        nameLabelView.text = "Hi, " + name! + "!"
+        
+        setupLayout()
+        
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
```

Comment:
I think with that line, If you moved the app to background then to foreground again this the particles emission will be doubled.
we can leave it for now but be carful when choosing which lifecycle function you are using